### PR TITLE
von Everec family

### PR DESCRIPTION
**Pull request by TMLCJLK**
Added von Everec family line and some related characters. There may be other characters to add to the family, but I will have to revisit the crypt to check it out.

Also, at this point they hold no titles during their history. Should they be a Novigrad family or hold a barony at Oxenfurt?

### CODE REVIEW

**Comment by JonStryker** [comment]
You should just use the "canon" trait. I mean many of the characters labelled "canon" are just mentioned in passing in the books or in the first two games. Olgierd has a whole DLC that revolves around him!

**Comment by TMLCJLK** [comment]
But since we are separating between canon (characters that appears in Andrzej Sapkowski's works) and semi-canon (characters that only appears in derivative works), even a main character from the games is a semi-canon.

**Comment by JonStryker** [comment]
Believe me, when I made all the characters I made sure to put a canon trait on anybody that is mentioned in either the books or the first two Witcher games (since the 3rd one wasn't out then) ;)

**Comment by TMLCJLK** [comment]
I believe you. But rquinio suggested adding a semi-canon trait to differentiate them on these terms.

**Comment by rquinio** [comment]
@JonStryker I've added this semi-canon trait with commit https://github.com/ajjr1996/Witcher/commit/36a636bcfe85fd3bbd694f03b59ecace91169a5f (and I've corrected some existing characters with https://github.com/ajjr1996/Witcher/commit/da4a52fe015b6f654eb5fa5e16d80438ced5fc01)
It shouldn't be too hard, since the person who adds it generally knows the source (and the wikis keeps track of this too). It's mostly for general documentation, to know where a character comes from, as I've read that Andrzej Sapkowski doesn't acknowledge the video games as canon, so we could end up with conflicts and a big headache in the future depending on the timeline of the Netflix series or future books.
It's also a way to indirectly explain "in-game" to the players that in the mod we consider books > games, so the dates are the ones from books etc., as there's been some recurring questions on that.

File: MOD/Witcher/history/characters/155000_special.txt [hunk 1]
```diff
@@ -63,21 +63,7 @@
...
-					}
-				}
-			}
-		}
-	}
-	1275.1.1 = {
-		death = yes
-		death_reason = death_dungeon
-	}
+	1274.6.1 = { death = yes }
```
Comment (rquinio) [inline MOD/Witcher/history/characters/155000_special.txt]: Ok that makes sense as "Left Síle de Tansarville to die." is the default preset in the "Witcher 3" (which can be considered "canon") according http://witcher.wikia.com/wiki/The_Witcher_3_decision_checklist

**Comment by rquinio** [comment]
I'm going to merge this, and synch from master, so that the W3 branch works with 2.8.1
More review can be done when we'll merge to master.

**Pull request merged**